### PR TITLE
server: Convey how to configure network for server

### PR DESCRIPTION
**Pull request by mdhorn**

### CODE REVIEW

File: scripts/add-login-issue.sh [hunk 1]
```diff
@@ -6,9 +6,12 @@ echo "Welcome to the Clear Linux* OS live image!
  * Documentation:     https://clearlinux.org/documentation
  * Community Support: https://clearlinux.org/community
 
+To configure the network run:
+  nmtui
+
 To install Clear Linux* OS onto this sytem please login as root,
```
Comment (reaganlo) [inline scripts/add-login-issue.sh]: Not part of this PR.. But I see a "sytem" typo in the vicinity :)

**Comment by mdhorn** [comment]
@reaganlo Update the 'system' typo
